src/main/kotlin/io/github/mkohm/demo/lcom/LowCohesion.kt

```diff
@@ -0,0 +1,19 @@
+package io.github.mkohm.demo.lcom
+
+class LowCohesion {
```

Comment:
Class `LocationServiceRepository` has low cohesion (LCOM = 0.78).  Consider extracting functionality into separate classes to follow the Single Responsibility Principle.

Property `number0` is used by `fun inc0()` 
Property `number1` is used by `fun inc1()` 
Property `number2` is used by `fun inc2()`